test/test_helpers/sidekiq.rb

```diff
@@ -3,6 +3,8 @@
 require 'sidekiq/testing'
 require 'sidekiq/lock/testing/inline'
 
+# Turn off Sidekiq logging which pollutes the CI logs
+Sidekiq::Logging.logger = nil
```

Comment:
Won't this affect non-CI test executions too?

Reply:
This is for test environment, so what do you want exactly?

Reply:
Full Sidekiq logs when we run the tests in development.

Reply:
Do you need that really?

Reply:
I would rather have them, but let's not make the discussion longer than it needs to be. You decide.

In case you want to make it CI-only, I think you can use:

```ruby
Sidekiq::Logging.logger = nil if ENV['CI']
```

Reply:
OK but they were not there before, it was introduced with the work on Circle CI.
Lets do it in another PR  so we can discuss about it